ui/shared/CopyToClipBoardButton.qml

```diff
@@ -44,6 +47,31 @@ Rectangle {
             onClick()
         }
     }
+
+    ToolTip {
+        id: toolTip
+        text: parent.textTooltip
+        parent: copyToClipboardButton
+    }
+
+    Timer {
+        id:showTimer
+        interval: 500
+        running: parent.showTooltip && !toolTip.visible
+        onTriggered: {
+            toolTip.visible = true;
+        }
+    }
+
+    Timer {
+        id:hideTimer
+        interval: 1500
```

Comment:
I feel like `1500` might be too quick to read?

Reply:
what about 2 or 2.5 sec?

Reply:
I'd need to test for myself, but it's surely better. Try to find an amount that leaves enough time to read the text